metadata.yaml

```diff
@@ -0,0 +1,10 @@
+name: glauth-operator
+display-name: glauth-operator
```

Comment:
Should the display name just be `GLAuth Operator`?

```suggestion
display-name: GLAuth Operator
```

Reply:
Sounds good, updated!